### ISSUE TITLE
iOS device not recognised. 

### ISSUE THREAD

**Issue by dekosan** [koush/vysor.io#843]
**Describe the bug**
Use of iPhone SE2, original lightning data cable. Android works OK. On Mac with high Sierra.

**You must attach a Vysor Bug Report Link**

Nothing happens...

**To Reproduce**
Connect iPhone

**Expected behavior**
"Trust computer" is not shown

**Screenshots**
Just startup screen

**Desktop (please complete the following information):**
MacOS 10.13.6

**iOS (please complete the following information):**
 - Device: iPhone SE2
 - OS: iOS 14.6
 - Vysor Bug Report Link: NA

**iOS Bluetooth Issues (optional):**
NA

**Additional context**

**Comment by koush** [comment]
What year iPhone is this? I’m actually unsure if iPhone se is even supported. Does it show up for screen recording in QuickTime?

**Comment by dekosan** [comment]
It’s an iPhone SE 2, MXD02ZD/A, which is not an old one. In screen recording it does not show up, in movie recording it DOES show up.

**Comment by koush** [comment]
Yes, movie recording is what I met. Can you try just closing/reopening Vysor (right click quit). Also, attach a bug report after it starts back up. You did not do that in the original post.

**Comment by dekosan** [comment]
https://gist.github.com/9870f9db5878a403bb14cb8b20d5d641 <https://gist.github.com/9870f9db5878a403bb14cb8b20d5d641>

**Comment by dekosan** [comment]
and again

https://gist.github.com/c1c164af1d6dea08878ddc1bdc1227eb <https://gist.github.com/c1c164af1d6dea08878ddc1bdc1227eb>

**Comment by AlucardX14** [comment]
iPhone SE2 here as well. To this day still not detected in Vysor app for broadcasting
![image](https://user-images.githubusercontent.com/33877126/191379882-eeb3c9a8-acd9-4ea1-a1e5-1be29d0d2b44.png)

**Comment by koush** [comment]
I think it's there, try scrolling that list.

**Comment by AlucardX14** [comment]
It is not, I've scrolled. Others in the App Store reviews have also mentioned having the same issue.
I'm on iOS 14.2 if that matters

**Comment by AlucardX14** [comment]
This is my jailbroken iPhone so I'm not updating it. I tested an iPhone XR on 14.6 and it works fine though.
I have to point out that my SE has had this issue since before being jailbroken too, so you can discard that as being the issue.

If there's anything I can do to help troubleshoot this issue, please let me know.

**Comment by AlucardX14** [comment]
It all comes down to the iOS version? Anything else that can be done?
